### DISCUSSION TITLE
Welcome to vue-final-modal Discussions!

### DISCUSSION THREAD

**Discussion by hunterliu1003** [vue-final/vue-final-modal#29] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sunil-dzango** [comment]
Hello, using the package for the first time in Vue 3. I followed the documentation and integrated into my project but I am having some rendering issues. There is no modal rendered, only the content on the top of the page. Did I miss something to integrate proper css?

**Reply by hunterliu1003** [reply]
You have to implement CSS yourself with vue-final-modal. Because vfm is an renderless component.

**Reply by sunil-dzango** [reply]
I see. I just figured out that. Thanks!
